constants.xml

```diff
@@ -200,6 +200,11 @@
         <RESUME_BLACKLIST_DELAY_IN_SECONDS>30</RESUME_BLACKLIST_DELAY_IN_SECONDS>
         <INCRDB_DSNUMS_WITH_STATEDELTAS>10</INCRDB_DSNUMS_WITH_STATEDELTAS>
     </recovery>
+    <server>
```

Comment:
The name `server` is a bit vague. Can it be just called `jsonrpc`?

Also, the option `ENABLE_SHARD_SERVER`. Are we sure about the name `SHARD_SERVER`?

Reply:
okay, how about `jsonrpc` and `ENABLE_STATUS_RPC`